modules/file-integrity-operator-installing-cli.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/file-integrity-operator-installation.adoc
+
+[id="installing-file-integrity-operator-using-cli_{context}"]
+= Installing the File Integrity Operator using the CLI
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Create a namespace object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-file-integrity
+----
+
+. Apply the file by running:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
+
+. Create the `OperatorGroup` object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: file-integrity-operator
+  namespace: openshift-file-integrity
+spec:
+  targetNamespaces:
+  - openshift-file-integrity
+----
+
+. Create the Subscription object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: file-integrity-operator-sub
+  namespace: openshift-file-integrity
+spec:
+  installPlanApproval: Automatic
+  name: file-integrity
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+. Verify the installation succeeded by inspecting the CSV file:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-file-integrity
+----
+
+At this point, the Operator should be up and running:
+
+[source,terminal]
+----
+$ oc get deploy -n openshift-file-integrity
+----
```

Comment:
We also get some PRs about the installation on restricted cluster. Could  we put a link here to indicate customer `for restricted cluster, please refer to https://docs.openshift.com/container-platform/4.6/operators/admin/olm-restricted-networks.html`